nixos/modules/services/security/oauth2_proxy_nginx.nix

```diff
@@ -0,0 +1,64 @@
+{ pkgs, config, lib, ... }:
+with lib;
+let
+  cfg = config.services.oauth2_proxy.nginx;
+in
+{
+  options.services.oauth2_proxy.nginx = {
+    proxy = mkOption {
```

Comment:
Are there cases where this can be something other than `config.services.oauth2_proxy.httpAddress`?

Reply:
@yorickvP

Reply:
yes, when the proxy runs on another system.